app.json

```diff
@@ -19,10 +19,20 @@
     "MANDRILL_USERNAME": "REDACTED",
     "MANDRILL_API_KEY": "REDACTED",
     "MANDRILL_DOMAIN": "heroku.com",
-    "GOOGLE_SECRET": {
```

Comment:
We can't remove ENV vars like this, it'll break the install of anyone who used `GOOGLE_SECRET` and `GOOGLE_KEY` in the past. We'll have to check for both inside the config code.

Reply:
Ouch, I went off of the previous question about removing.

Reply:
I guess it makes sense to not break this. Should be able to add new env vars in Heroku under the new names and copy over values.

It's nice to prevent the headache in the first place though.